### PR TITLE
Pass on PRC117F

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Lazy evaluation
- Formatting

### CODE REVIEW

File: addons/sys_io/fnc_server.sqf [hunk 1]
```diff
@@ -31,7 +31,7 @@ DFUNC(connectionFnc) = {
                         if ((missionNamespace getVariable [QGVAR(notConnectedTime), -15]) + 30 < time ) then {
                             GVAR(notConnectedTime) = time;
                             private _warning = format ["<t color='#FF8021'>WARNING!</t><br/> %1", localize LSTRING(acreNotConnected)];
-                            [[_warning, 1.5]] call CBA_fnc_notify;
+                            [[_warning, 2]] call CBA_fnc_notify;
```
Comment (jonpas) [inline addons/sys_io/fnc_server.sqf]: No, keep it at 1.5. 2 has trouble going off-screen, it's just too big.

File: addons/sys_io/fnc_server.sqf [hunk 2]
```diff
@@ -46,7 +46,7 @@ DFUNC(connectionFnc) = {
             } else {
                 LOG("PIPE OPENED!");
                 if (GVAR(hasErrored) && isMultiplayer) then {
-                    [[format ["<t color='#2B7319'>%1</t>", localize LSTRING(recoveredClosedPipe)], 1.5]] call CBA_fnc_notify;
+                    [[format ["<t color='#2B7319'>%1</t>", localize LSTRING(recoveredClosedPipe)], 2]] call CBA_fnc_notify;
```
Comment (jonpas) [inline addons/sys_io/fnc_server.sqf]: Same here, merge issue?

File: addons/sys_prc117f/menus/fnc_formatText.sqf [hunk 3]
```diff
@@ -51,13 +51,15 @@ while { _result != -1 && _iter < 5} do {
...
-    (
-        [
-            _key,
-            _value
-        ] call FUNC(formatChannelValue)
-    )
+    _text = [
+        _text,
+        _replacementValue,
+        (
```
Comment (jonpas) [inline addons/sys_prc117f/menus/fnc_formatText.sqf]: This is pretty damn ugly, I prefer:
```suggestion
        [_key, _value] call FUNC(formatCHannelValue)
```

**Review by jonpas — CHANGES_REQUESTED** [review]

File: addons/sys_io/fnc_server.sqf (hunk 1 above)
Comment (TheMagnetar) [reply addons/sys_io/fnc_server.sqf]: Merge issue...

File: addons/sys_io/fnc_server.sqf (hunk 2 above)
Comment (TheMagnetar) [reply addons/sys_io/fnc_server.sqf]: Merge issue...

**Review by jonpas — APPROVED** [review]

**Pull request merged**